server/src/test/java/org/apache/druid/server/coordinator/duty/CompactSegmentsTest.java

```diff
@@ -640,7 +655,12 @@ public void testRunMultipleCompactionTaskSlots()
     final CoordinatorRunStats stats = doCompactSegments(compactSegments, 3);
     Assert.assertEquals(3, stats.get(Stats.Compaction.AVAILABLE_SLOTS));
     Assert.assertEquals(3, stats.get(Stats.Compaction.MAX_SLOTS));
-    Assert.assertEquals(3, stats.get(Stats.Compaction.SUBMITTED_TASKS));
+    // Native takes up 1 task slot by default whereas MSQ takes up all available upto 5.
```

Comment:
I am not sure if this comment properly explains what is being asserted next.
We are asserting 3 for native but 1 for msq.

Reply:
Expanded on it now.